paper.md

```diff
@@ -0,0 +1,178 @@
+
+# Statement of Need
+There are many great free online course materials on deep learning, 
+for example: fast.ai's Practical Deep Learning for Coders, MORE ONLINE COURSES
+
+Nonetheless, these resources are often not available open-source and can thus not be easily adapted to the students' needs. 
+Also, the amount and diversity of the existing online courses is overwhelming, but they are generally not targeted towards academic researchers.
+What works well for researchers is to both make them familiar with the key concepts, and also let them 
+practice with how to implement it. Eventually resulting in a feeling of 'I can do this myself'. 
+The key to getting there is live coding: Before the course learners have to setup a working environment on their own computer.
+During the course learners type in the commands that are explained by the instructor on their own computer.
+This ensures that learners master the programmatic implementation of deep learning at the end of the course. 
+
+Researchers can often only free a limited amount of time (maximum 5 consecutive days), since they are so involved in their daily work.
+To accomplish this, we created a lesson that can be taught in 2 consecutive days or 4 mornings.
+
+From the feedback we gathered from students enrolled in our workshops and instructors not involved in our project we see that indeed there is a 
+need for a low-threshold lesson that lets researchers take the first steps in the field of deep learning.
+
+# Instructional design
+This lesson material was designed using the concepts from the Carpentries Curriculum Development Handbook (REFERENCE https://cdh.carpentries.org/).
+Most importantly, we used 'backward design': We started with identifying learning objectives: the core skills and concepts that learners need to acquire as a result of the lesson.
+Then, these learning objectives were used to create exercises that test whether the objectives are met.
+Eventually, the content is written in such a way that it matches what learners need to learn to successfully apply these skills.
+
+Furthermore, the lesson is built around live coding. 
+The lesson is built up of small blocks. In each block first the instructor demonstrates how to do something,
+and students follow along on their own computer. Then, the students work independently on exercises individually
+or in groups to test their skills.
+
+The lesson material is built in the new Carpentries lesson template (REFERENCE). 
+This makes the lesson material both suited as a self-study resource, a syllabus to share with students after the workshop,
+as well as lesson material for the instructor. With the 'instructor view' we even include instructor notes 
+at the level of the lesson content.
+
+The lesson is split up in a general introduction, and 3 episodes that cover 3 distinct increasingly more complex deep learning problems.
+Each of the deep learning problems is approached using the same 10-step deep learning workflow.
```

Comment:
List the ten steps here? Or is that spoilers? 😆

Reply:
Hmmm, the instructions for submitting to JOSE state clearly that the paper should not go into details on the content. The lesson itself is the resource for that. But we should link to the 10-step deep learning workflow: https://carpentries-incubator.github.io/deep-learning-intro/1-introduction.html#deep-learning-workflow See ddf9378b865756f0ff7d914e0b37c3ea5304282f